### ISSUE TITLE
Enhancement: Update Authentication Page for Chimoney API Documentation

### ISSUE THREAD

**Issue by phyleria** [Chimoney/chimoney-community-projects#374]
### Description

We need an updated detailed Authentication page that explains how to authenticate requests to the Chimoney API. 

[Current Page
](https://chimoney.readme.io/reference/authentication)
This page should cover:

- **API Key Authentication**: Provide clear instructions on how to use API keys to authenticate requests, including proper use of the Authorization header.
- **Security Best Practices**: Highlight important security tips like storing API keys in environment variables, keeping keys secret, and rotating them periodically.
- **Error Handling**: Explain the common authentication-related errors (e.g., 401 Unauthorized) and how to resolve them.
- **Example Requests**: Include example curl commands or code snippets showing how to authenticate API requests.
- **Response Format**: Show sample responses for both successful authentication and common authentication errors.

## Acceptance Criteria:

- The page must provide a clear, concise explanation of API authentication.
- Include at least one example of an authenticated API request (using curl or another tool).
- Highlight security best practices and error handling.
- Ensure all technical terms are well-explained, and links to further documentation are provided.

### Ready to Work?

- [ ] I want to work on this enhancement

**Comment by ayan-joshi** [comment]
Hey @phyleria , I want to work on this enhancement and I'm ensuring of a better result too. 
Thanks for your time !

**Comment by phyleria** [comment]
Ok, I am assigning it to you right now

**Comment by ayan-joshi** [comment]
@phyleria This I can do right ?

**Comment by phyleria** [comment]
Hi @ayan-joshi 

Will you still be able to work on this?

**Comment by oluwatobi2001** [comment]
Hi
 I am interested in working on this. Thank you

**Comment by ayan-joshi** [comment]
> Hi @ayan-joshi
> 
> Will you still be able to work on this?

@phyleria Yes I just wanted to know where in the folder this need to be edited and change ?

**Comment by ayan-joshi** [comment]
> @phyleria Hey , as it says we need to update the authentication page , where I can find the previous page code as it will be with you on private repo , so may I write a whole new page then?

@phyleria have a look at this please

**Comment by phyleria** [comment]
Hi @ayan-joshi,  

The API docs ReadMe isn't currently open to contributors. Could you kindly create a new markdown file with the current contents of the Authentication page, including your updates and submission? Once done, please submit it within the **articles** folder.  

Does this sound clear? Let me know if you need any further clarification—happy to help!

**Comment by ayan-joshi** [comment]
> Hi @ayan-joshi,
> 
> The API docs ReadMe isn't currently open to contributors. Could you kindly create a new markdown file with the current contents of the Authentication page, including your updates and submission? Once done, please submit it within the **articles** folder.
> 
> Does this sound clear? Let me know if you need any further clarification—happy to help!

Okay thanks , I'll follow this @phyleria

**Comment by ayan-joshi** [comment]
@phyleria quite busy with college , sorry for the delay I'll complete this and open the pr in the next week as early as possible

**Comment by phyleria** [comment]
ok @ayan-joshi ✅
